accounting/bank/migrations/0001_initial.py

```diff
@@ -0,0 +1,59 @@
+                ('name', models.CharField(help_text='The official name of the bank.', max_length=80)),
+                ('address', models.ForeignKey(help_text='The address of this user.', on_delete=django.db.models.deletion.CASCADE, related_name='banks', to='account.Address')),
+            ],
+            options={
+                'verbose_name': 'Bank',
+                'verbose_name_plural': 'Banks',
+            },
+        ),
+        migrations.CreateModel(
+            name='BankAccount',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, help_text='The universally unique identifier for this bank account.', verbose_name='UUID')),
+                ('currency', djmoney.models.fields.CurrencyField(default='EUR', help_text='The currency expected to be held in this bank account.', max_length=3)),
+                ('type', models.CharField(choices=[('checking', 'Checking Account'), ('savings', 'Savings Account')], help_text='Whether this is a checking or savings account.', max_length=30)),
+                ('iban', localflavor.generic.models.IBANField(blank=True, help_text='The unique International Bank Account Number of the provider.', include_countries=None, max_length=34, null=True, use_nordea_extensions=False)),
+                ('bic', localflavor.generic.models.BICField(blank=True, help_text='The 11-character SWIFT / BIC (Business Identifier Code) code used to identify a bank or financial institution globally.', max_length=11, null=True)),
+                ('abn', localflavor.au.models.AUBusinessNumberField(blank=True, help_text='The 11-character Australian Business Number used to identify business entities in Australia.', max_length=11, null=True)),
+                ('bsb', accounting.bank.fields.AUBankStateBranchField(blank=True, help_text='The 6-character Bank State Branch code used as a bank identifier in Australia.', max_length=6, null=True)),
+                ('vat', vies.models.VATINField(blank=True, help_text='The value added tax identification number used in and required by some countries. This can also be used to store a general sales tax (GST) identification number.', max_length=14, null=True)),
+                ('account_number', models.CharField(blank=True, help_text='The bank account number used to help identify the account. Required for only some countries.', max_length=30, null=True)),
+                ('routing_number', models.CharField(blank=True, help_text='The bank account routing number used to help identify the account. Required for only some countries.', max_length=30, null=True)),
+                ('bank', models.ForeignKey(help_text='The bank in which this bank account is opened.', on_delete=django.db.models.deletion.CASCADE, related_name='bank_accounts', to='bank.Bank')),
```

Comment:
@UmanShahzad This sounds like it's the accounting app's business to open bank accounts :) I'd probably change this to "The bank to which this bank account belongs."

Reply:
:+1: